src/__tests__/parser/expression.spec.ts

```diff
@@ -1459,4 +1459,33 @@ describe('SonicWeave expression evaluator', () => {
     const zero = parseSingle('gcd()');
     expect(zero.toString()).toBe('0');
   });
+
+  it('can calculate the step string for Lydian', () => {
+    const pattern = evaluateExpression(
+      '9/8;81/64;729/512;3/2;27/16;243/128;2/1;stepString()'
+    );
+    expect(pattern).toBe('LLLsLLs');
+  });
+
+  it('can calculate the step string for pseudo-Ionian in 8edo', () => {
+    const pattern = evaluateExpression(
+      '2\\8;4\\8;3\\8;5\\8;7\\8;9\\8;8\\8;stepString()'
+    );
+    expect(pattern).toBe('xxμxxxμ');
```

Comment:
Seeing this written out like this makes me inclined to favor uppercase letters instead. This would be `'XXμXXXμ'` instead.